docs/project-generation-options.rst

```diff
@@ -63,6 +63,12 @@ js_task_runner:
     1. None
     2. Gulp_
 
+cloud_provider:
+    Select a cloud provider for static files. The choices are:
```

Comment:
I suggest extending this to media as well. They mean 2 different things in the Django space:

```suggestion
    Select a cloud provider for static & media files. The choices are:
```